packages/language-server/src/plugins/typescript/service.ts

```diff
@@ -640,6 +737,21 @@ async function createLanguageService(
             }
         }
 
+        const svelteConfigDiagnostics = checkSvelteInput(parsedConfig);
+        if (docContext.reportConfigError) {
```

Comment:
Shouldn't we only do this when there's actual diagnostics?
```suggestion
        if (docContext.reportConfigError && svelteConfigDiagnostics.length) {
```